heal/vlmd/validate/json_validator.py

```diff
@@ -8,7 +9,7 @@
 from heal.vlmd.config import ALLOWED_SCHEMA_TYPES
 from heal.vlmd.validate.utils import get_schema
 
-logger = get_logger("JSON_VALIDATOR", log_level="debug")
+logger = get_logger("json-validator", log_level="debug")
 
```

Comment:
Here is another example where the log_level in a child method could be removed and controlled from a higher level method.